load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1365,6 +1365,12 @@ func (s *relaysSuite) querySharedParams() {
 	s.Helper()
 
 	deps := depinject.Supply(s.txContext.GetClientCtx())
+
+	queryNodeRPCURL := testclient.CometLocalWebsocketURL
```

Comment:
Isn't the address dependant on the environment that's under test (e.g. Localnet vs devnet)?